# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: netwerk/base/SimpleChannel.h
# Commit: 74f99033251c
# Full Hash: 74f99033251cd6c36eb6b0b5a4713eb6b5c793fa
# Author: Jonathan Kingston <jkt@mozilla.com>
# Date: 2019-02-13 09:48:26
# Regressor Bug: 1520868
# File Overlap Count: 1
# Description:
#   Bug 1520868 - Replacing AsyncOpen2 with AsyncOpen always r=valentin
#   
#   Replacing js and text occurences of asyncOpen2
#   Replacing open2 with open
#   
# ==============================================================================

diff -r 3378fd5f4f22 -r 74f99033251c netwerk/base/SimpleChannel.h
--- a/netwerk/base/SimpleChannel.h	Tue Feb 12 17:58:19 2019 +0100
+++ b/netwerk/base/SimpleChannel.h	Tue Feb 12 16:08:25 2019 +0000
@@ -73,8 +73,8 @@
 
 /**
  * Creates a simple channel which wraps an input stream created by the given
- * callbacks. The callbacks are not called until the underlying AsyncOpen2 or
- * Open2 methods are called, and correspond to the nsBaseChannel::StartAsyncRead
+ * callbacks. The callbacks are not called until the underlying AsyncOpen or
+ * Open methods are called, and correspond to the nsBaseChannel::StartAsyncRead
  * and nsBaseChannel::OpenContentStream methods of the same names.
  *
  * The last two arguments of each callback are the created channel instance,